tests/lldp/test_lldp_syncd.py

```diff
@@ -148,9 +148,8 @@ def verify_lldp_entry(db_instance, interface):
 
 
 def verify_lldp_table(duthost):
-    # based on experience, eth0 shows up at last
     output = duthost.shell("show lldp table")["stdout"]
-    if "eth0" in output:
+    if "Total entries displayed" in output:
```

Comment:
Will the lldp table doesn't have an entry, will it have log like 'Total entries displayed: 0'?

Reply:
It is possible, but in this function, it means the command of show lldp table works, no traceback, no error.
If the content is empty, the next table content check will report the unexpected problem if it has.

Reply:
Got it, thanks for explaining.